DSA/binary_search.js

```diff
@@ -0,0 +1,122 @@
+// Binary Search
+
+// Explaination : 
+// Binary Search is applied to a sorted array or list. It's a divide-and-conquer algorithm that efficiently finds the position of a target element in the array. 
+// Binary Search is a highly efficient algorithm for searching in sorted data, with a time complexity of O(log n), where n is the size of the array.
+
+function binarySearch(arr, target) {
```

Comment:
Can you please follow JS Doc standards for comments?
https://jsdoc.app/about-getting-started.html

Reply:
Sure,  I'll work on it asap